### DISCUSSION TITLE
PID Autotune

### DISCUSSION THREAD

**Discussion by brett0712** [Jyers/Marlin#232] [Q&A] [answered]
When I run PIDAutotune  for either the nozzle or bed, the confirmation notice freezes and I cannot clear it. I have to power off the printer in order to get rid of the notification. 

I am still able to run commands through terminal however the knob becomes nonresponsive. I don't think it is a total lockup as the temperature readouts still update correctly.

I read somewhere in the past (I think on Reddit) that there was similar behavior with notifications for filament change either not clearing or delayed and it was recommended that the printer be restored to the default firmware. I did this and it made no difference. 

Has anyone else encountered this?

Update: I have 3 Ender 3V2s all configured the exact same exhibiting the same behavior.

**Comment by kentsangcanada** [comment]
I can confirm this happens to the current version


=================

Send: M303 E0 S200 U1
Recv: //action:notification PID Autotune
Recv: PID Autotune start
[...]
Recv:  bias: 76 d: 76 min: 197.73 max: 207.11 T:198.89 /0.00 B:24.41 /0.00 @:76 B@:0
[...]
Recv:  bias: 73 d: 73 min: 197.27 max: 203.33 T:199.09 /0.00 B:24.41 /0.00 @:73 B@:0
[...]
Recv:  bias: 73 d: 73 min: 197.27 max: 202.81 Ku: 33.56 Tu: 32.86
Recv:  Classic PID
Recv:  Kp: 20.13 Ki: 1.23 Kd: 82.70
[...]
Recv:  bias: 73 d: 73 min: 197.27 max: 203.26 Ku: 31.03 Tu: 33.50
Recv:  Classic PID
Recv:  Kp: 18.62 Ki: 1.11 Kd: 77.97
[...]
Recv:  bias: 73 d: 73 min: 197.27 max: 202.81 Ku: 33.56 Tu: 33.02
Recv:  Classic PID
Recv:  Kp: 20.13 Ki: 1.22 Kd: 83.11
Recv: PID Autotune finished! Put the last Kp, Ki and Kd constants from below into Configuration.h
Recv: #define DEFAULT_Kp 20.13
Recv: #define DEFAULT_Ki 1.22
Recv: #define DEFAULT_Kd 83.11
Recv: //action:notification Ender-3 V2 Ready.
Recv: ok
[...]



![IMG_3940](https://user-images.githubusercontent.com/6929546/110875212-3414f180-82a3-11eb-9aef-5cadc7276df2.JPG)

**Comment by Jyers** [comment, marked as answer]
This should be fixed in the latest source and will be part of the next release.
